packages/contracts-bedrock/contracts/test/invariants/L2OutputOracle.t.sol

```diff
@@ -0,0 +1,89 @@
+pragma solidity 0.8.15;
+
+import { L2OutputOracle } from "../../L1/L2OutputOracle.sol";
+import { L2ToL1MessagePasser } from "../../L2/L2ToL1MessagePasser.sol";
+import { Predeploys } from "../../libraries/Predeploys.sol";
+import { Test } from "forge-std/Test.sol";
+
+contract L2OutputOracle_Invariants is Test {
+    // Global
+    address multisig = address(512);
+
+    // Test target
+    L2OutputOracle oracle;
+
+    L2ToL1MessagePasser messagePasser =
+        L2ToL1MessagePasser(payable(Predeploys.L2_TO_L1_MESSAGE_PASSER));
+
+    // Constructor arguments
+    address internal proposer = 0x000000000000000000000000000000000000AbBa;
+    address internal owner = 0x000000000000000000000000000000000000ACDC;
+    uint256 internal submissionInterval = 1800;
+    uint256 internal l2BlockTime = 2;
+    uint256 internal startingBlockNumber = 200;
+    uint256 internal startingTimestamp = 1000;
+
+    // Test data
+    uint256 initL1Time;
+
+    // Advance the evm's time to meet the L2OutputOracle's requirements for proposeL2Output
+    function warpToProposeTime(uint256 _nextBlockNumber) public {
+        vm.warp(oracle.computeL2Timestamp(_nextBlockNumber) + 1);
+    }
+
+    function setUp() public virtual {
+        // By default the first block has timestamp and number zero, which will cause underflows
+        // in the tests, so we'll move forward to these block values.
+        initL1Time = startingTimestamp + 1;
+        vm.warp(initL1Time);
+        vm.roll(startingBlockNumber);
+
+        // Deploy the L2OutputOracle and transfer owernship to the proposer
+        oracle = new L2OutputOracle(
+            submissionInterval,
+            l2BlockTime,
+            startingBlockNumber,
+            startingTimestamp,
+            proposer,
+            owner
+        );
+    }
+
+    /**
+     * INVARIANT: The block number of the output root proposals should monotonically increase.
```

Comment:
Do you mind updating this to use the custom tag from the other PR for consistency?

Reply:
Will do after dinner! Must've not restacked- thanks for the spot 😄